packages/mdc-tab/foundation.js

```diff
@@ -84,16 +96,20 @@ class MDCTabFoundation extends MDCFoundation {
 
   /**
    * Activates the Tab
+   * @param {!ClientRect=} previousIndicatorClientRect
```

Comment:
I forget my closure syntax...but can previousIndicatorClientRect be undefined?

Reply:
It's optional. The `=` makes it optional (undefined) where as the `?` makes it nullable. We want it to be optional in this case.